db/post_migrate/20210420162817_mark_badges_for_beginners.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class MarkBadgesForBeginners < ActiveRecord::Migration[6.0]
```

Comment:
In addition to this migration I think you need to set up these beginner badges in the badge fixture

https://github.com/discourse/discourse/blob/86ef3b4091876ab9a884fa7f4352f436975eed34/db/fixtures/006_badges.rb

I am fairly sure this is run when the Discourse site is first set up

Reply:
Actually now that I think about it, is there any reason you even need a new column? Can't you just rely on the badge_grouping_id of `BadgeGrouping::GettingStarted` which is already set up?

Reply:
Unfortunately, we can’t rely on the `badge_grouping_id` only.

Now we are skipping all the badges from the **Getting Started** group _excluding_ the <kbd>New User of the Month</kbd> badge _plus_ the <kbd>Welcome</kbd> badge (which is in the **Community** group).

And we want to be able to skip certain badges without moving them between groups.

Reply:
Ah yes makes sense, I guess the fixtures just need to be updated?

Reply:
Yeah, I've updated the fixture.